src/main/java/spireMapOverhaul/util/ActUtil.java

```diff
@@ -0,0 +1,11 @@
+package spireMapOverhaul.util;
+
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+
+public class ActUtil {
+    public static int getRealActNum() {
```

Comment:
I extracted this out of AbstractZone because I wanted it to stay consistent if we had to modify this later.

For example, I'm worried that there's a bug with this if you're in endless mode and decide to go to act 4+. Unless `Settings.endless` gets set to false when you enter act 4, won't this say you're in act 1 again? Didn't have time to test this, but having the logic in one place means we only need to fix one place if there are any bugs like that.